src/component/Home.jsx

```diff
@@ -3,6 +3,23 @@ import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
```

Comment:
Connect MUI with React router also in all `<Link>` components in project. The goal is to render MUI components with predefined appearance and use RouterLink only in `component` prop.

MyMenu.jsx
`Typography + RouterLink => Link`
`Button + RouterLink => Button`

Plans.jsx
`Typography + RouterLink => Link`